easypost/billing.py

```diff
@@ -0,0 +1,81 @@
+from typing import (
+    Any,
+    Dict,
+    List,
+    Optional,
+)
+
+from easypost.easypost_object import convert_to_easypost_object
+from easypost.error import Error
+from easypost.requestor import (
+    RequestMethod,
+    Requestor,
+)
+from easypost.resource import (
+    CreateResource,
+    Resource,
+)
+
+
+class Billing(CreateResource, Resource):
+    @classmethod
+    def fund_wallet(cls, amount: str, primary_or_secondary: str = "primary", api_key: Optional[str] = None) -> bool:
+        """Fund your EasyPost wallet by charging your primary or secondary payment method on file."""
+        [endpoint, payment_method_id] = Billing._get_payment_method_info(primary_or_secondary=primary_or_secondary)
+
+        requestor = Requestor(local_api_key=api_key)
+        url = f"{endpoint}/{payment_method_id}/charges"
+        wrapped_params = {"amount": amount}
+        requestor.request(method=RequestMethod.POST, url=url, params=wrapped_params)
+
+        # Return true if succeeds, an error will be thrown if it fails
+        return True
+
+    @classmethod
+    def delete_payment_method(cls, primary_or_secondary: str, api_key: Optional[str] = None) -> bool:
+        """Delete a payment method."""
+        [endpoint, payment_method_id] = Billing._get_payment_method_info(primary_or_secondary=primary_or_secondary)
+
+        requestor = Requestor(local_api_key=api_key)
+        url = f"{endpoint}/{payment_method_id}"
+        requestor.request(method=RequestMethod.DELETE, url=url)
+
+        # Return true if succeeds, an error will be thrown if it fails
+        return True
+
+    @classmethod
+    def retrieve_payment_methods(cls, api_key: Optional[str] = None, **params) -> Dict[str, Any]:
+        """Retrieve all payment methods."""
```

Comment:
Nit: `all` alludes to the fact that there are `many` which in this case there are not. I'd just remove `all` because `methods` is already plural implying multiple
```suggestion
        """Retrieve payment methods."""
```

Reply:
fixed